tools/run_tests/generated/tests.json

```diff
@@ -4813,6 +4813,30 @@
     ], 
     "uses_polling": true
   }, 
+  {
+    "args": [], 
+    "benchmark": false, 
+    "ci_platforms": [
+      "linux", 
+      "mac", 
+      "posix", 
+      "windows"
```

Comment:
Not sure it matters but I suspect this will not run under windows?

Reply:
good point, restricted to linux only - this test is indeed only meant for linux